eras/shelley/impl/src/Cardano/Ledger/Shelley/Utils.hs

```diff
@@ -0,0 +1,10 @@
+module Cardano.Ledger.Shelley.Utils 
+  ( Split(..)
+  )
+where
+
+import Cardano.Ledger.Coin (Coin)
+
+class Split v where
```

Comment:
This class was only used in tests before. No change to PParams should affect that. So, why do you need to move this class into `cardano-ledger-shelley`?